docs/lsm/quickstart/quickstart.rst

```diff
@@ -88,15 +92,15 @@ This folder contains a project.yml, which looks like this:
 
     Additional explanation of each field can be found on the quickstart.
 
-5. Change the <token> in the repo url to the credentials to the package repository (see Prerequisites section).
+5. Change the <token> in the repo url to the credentials of the package repository (see :ref:`Prerequisites section<lsm_quickstart_prerequisites>`).
```

Comment:
I think "credentials to" is actually correct, but I'm not sure.

Reply:
You are right.

Reply:
You are right.